StructureDefinition/NHSDigital-Organization.StructureDefinition.xml

```diff
@@ -40,6 +56,17 @@
       <min value="1" />
       <mustSupport value="true" />
     </element>
+    <element id="Organization.identifier:odsOrganisationCode">
+      <path value="Organization.identifier" />
+      <sliceName value="odsOrganisationCode" />
```

Comment:
Similarly, could we also get an agreed naming convention for slices so these are consistent across resources?

Reply:
Can you please show further what you mean?

Reply:
It's just here we have a slice on identifier for the ODS code, but, as with ValueSets/CodeSystems etc. we should probably agree how these should be named so we do not have slices using different conventions e.g. camelCase, prefixed by source of the slice (programme), with name matching the resource in the slice, if this is a ValueSet binding/Extension. For example the NHSBSA prescription type extension slice would then be named epsEnglandPrescriptionType (or something similar)